src/harness/certs/revoke_and_generate_crl.sh

```diff
@@ -2,53 +2,79 @@
 
 mkdir -p crl
 
-function blacklist_certificate()
+function revoke_certificate()
 {
-  #Argument1: Name of the certificate to be  blacklisted.
-  #Argument2: Intermediate CA to generate CRL.
+  #Argument1: Name of the certificate to be revoked.
 
+  #Fetch the Common Name to find out the issuer
+  CN=`openssl x509 -issuer -in $1 -noout | sed 's/^.*CN=//'`
+
+  if [ "$CN" == "WInnForum RSA CBSD CA-1" ]; then
+      CA=cbsd_ca
```

Comment:
This file uses a 2-space indent for nested blocks.  Please maintain that.

Reply:
Fixed the indent issue